packages/core/admin/package.json

```
@@ -28,6 +28,7 @@
     "develop:webpack": "cross-env NODE_ENV=development webpack serve --config webpack.config.dev.js --progress profile",
     "prepublishOnly": "yarn build",
     "build": "rimraf build && node ./scripts/build.js",
+    "mesure:webpack-build": "rimraf build && cross-env MESURE_BUILD_SPEED=true node ./scripts/build.js",
```

What do you think of reversing the name? Usually they are all written like: `[task]:[flavor]`, so this would become:

```suggestion
    "build:measure": "rimraf build && cross-env MESURE_BUILD_SPEED=true node ./scripts/build.js",
```